eq-author-api/schema/resolvers/base.js

```diff
@@ -71,6 +71,34 @@ const Resolvers = {
       ctx.repositories.Section.remove(args.input.id),
     undeleteSection: (_, args, ctx) =>
       ctx.repositories.Section.undelete(args.input.id),
+    createSectionIntroduction: (
+      _,
+      { input: { sectionId, introductionContent, introductionTitle } },
+      ctx
+    ) =>
+      ctx.repositories.Section.update({
```

Comment:
Updating the content and title seems pointless here.

Reply:
It's there to support the undo action without needing to make a new mutation for it. This way we can just re-create it with whatever's in the toast payload when they hit undo.

Reply:
I guess we can remove this when we remove undo